test/runtests.jl

```diff
@@ -29,6 +30,11 @@ using ChangesOfVariables: ChangesOfVariables
 using InverseFunctions: InverseFunctions
 using LazyArrays: LazyArrays
 
+# TODO(mhauru) Remove this once Enzyme is fixed?
```

Comment:
This is the default and shouldn't be needed now

Reply:
Removed